sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuBatchScanExec.scala

```diff
@@ -266,7 +266,7 @@ object GpuCSVScan {
         meta.willNotWorkOnGpu("GpuCSVScan does not support parsing timestamp types. To " +
           s"enable it please set ${RapidsConf.ENABLE_CSV_TIMESTAMPS} to true.")
       }
-      if (parsedOptions.zoneId.normalized() != GpuOverrides.UTC_TIMEZONE_ID) {
+      if (!TypeChecks.areTimestampsSupported(parsedOptions.zoneId.normalized())) {
```

Comment:
`areTimestampsSupported` needs to do the normalization internally to properly compare arbitrary timezone IDs passed to it.  It should not be the caller's responsibility to normalize first, or we risk someone forgetting to normalize.